test/verify/check-menu

```diff
@@ -44,6 +45,32 @@ class TestMenu(MachineCase):
         b.click('#about button[data-dismiss="modal"]')
         b.wait_popdown("about")
 
+        # Test session timeout
+        time.sleep(10)
+        with b.wait_timeout(1):
+            b.wait_not_visible("#session-timeout-dialog")
+            b.wait_visible("#filter-menus")
+            b.enter_page("/system")
+            b.mouse("#system_information_hardware_text", "mousemove", 24, 24)
+            b.switch_to_top()
+            time.sleep(34)
+            b.wait_visible("#session-timeout-dialog")
+            if int(b.text("#session-timeout-dialog #timeout-message").split()[-2]) < 20:
+                raise Exception("Unexpected remaining time in session")
```

Comment:
`self.assertGreater()` please.  But also, please give this a bit more leeway than 6s for robustness.